ament_lint_cmake/ament_lint_cmake/cmakelint.py

```diff
@@ -78,7 +78,7 @@
 """
 _DEFAULT_CMAKELINTRC = os.path.join(os.path.expanduser('~'), '.cmakelintrc')
 
-class _CMakeLintState(object):
+class _CMakeLintState:
```

Comment:
This file is an external project. Therefore I don't think we should modify it.

Reply:
oh I missed that, thanks for catching it https://github.com/ament/ament_lint/pull/83/commits/abefe7dfe39d8ee20e2ff04eb359210e79bc8128